public/js/src/bridge/copy_shipment_modal.tag

```diff
@@ -45,10 +45,16 @@
 
   saveShipment(evt) {
     var convertedShipment = utils.convertShipment(self.shipment);
-    // Build Shipment
-    self.loading = true;
+    
+    if ((convertedShipment.main.name + '-' + convertedShipment.environment.name).length > 63) {
+        RiotControl.trigger('flash_message', 'error', 'The combination of shipment and name are too long. Must be less than 63 chars.', 30000);
```

Comment:
The context of name in the error message doesn't make much sense. Also, we aren't limited on characters in the error message, so we should spell out characters. Here's a suggestion: 

```
The combination of Shipment and Environment as a name is too long. It must be less than 63 characters.
```